static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -277,31 +276,6 @@ const startPinterest = (): void => {
     }
 };
 
-const initialiseEmail = (): void => {
-    // Initalise email embedded in page
-    initEmail();
-
-    // Initalise email forms in iframes
-    Array.from(document.getElementsByClassName('js-email-sub__iframe')).forEach(
-        el => {
-            const iframe: HTMLIFrameElement = (el: any);
-
-            initEmail(iframe);
-        }
-    );
-
-    // Listen for interactive load event and initalise forms
-    bean.on(window, 'interactive-loaded', () => {
```

Comment:
I'm not sure what this is for - do you know?

Reply:
I'm not sure actually, but without `email.js` there won't be a `initEmail()` method for it to use so it could be worth some investigation? It looks like it's to initialise emails inside interactives, and `.guInteractive` doesn't appear in Frontend itself anywhere else.

Reply:
It looks like it's used as a class on the container element of a lot of interactives (did a Guardian github search). I wonder how many of those have email signups - perhaps it's just for those fancy sign up pages?

Reply:
Did a search for js-email-sub__iframe and that had a few hits as well - take a look.

Reply:
Looking at how and where this is used, I think it should all still be fine. The logic to handle what this form was doing previously is now all done either inside the iframe or using the Scala code I've edited here. Anywhere that the iframe is used it should still be effective. I've tested a couple of the interactives sign up pages, and then there were a couple of old repos that no longer have sign up pages (e.g. NHS emails), and they all still function as intended. 

In general I think that by removing the reliance on that single Frontend script, the iframes should be _more_ reusable across the platforms, as we're no longer requiring that to be there for the functionality to work.